designs/skully/src/uppermouth.mjs

```diff
@@ -125,13 +125,13 @@ export const uppermouth = {
     paths.front1 = new Path()
       .move(points.pointM1)
       .curve_(points.pointM1Cp1, points.point1)
-      .setText(complete ? '(16)' : '', textAttribute)
+      .setText(complete ? 'mouthTop' + '(16)' : '', textAttribute)
```

Comment:
This will break translation, since the text will be added as a single `mouthTop(16)` string.

Instead we should add the text in 2 steps:

```js
.addText(complete ? 'skully:mouthTop' : '')
.addText(complete ? '(16)' : '')
```

And then add `mouthTop` to `i18n/en.json` in the `s` section.

That should ensure it gets translated